### ISSUE TITLE
Need help confirming proper understanding of 2.4.3 Focus Order

### ISSUE THREAD

**Issue by A11yTea** [w3c/wcag#4895]
Hi everyone,
This is a super simple issue hopefully I can get some quick opinions to settle this debate.

### Situation:
At the bottom of a page there are two buttons that are styled in a way that the first button "Delete profile" is on the left side of the screen and the "Cancel" button is on the right side of the screen. They are both at the same height. 

<img width="522" height="46" alt="Screenshot showing the buttons described above" src="https://github.com/user-attachments/assets/33909a80-68f4-4341-b427-deafa7dd380a" />

### Problem:
The "Cancel" button is focused before the "Delete profile" despite the visual order being the contrary.

### Our Opinions:
Some believe that that this is a classic failure of focus order while others believe it would be 3.2.3: Consistent Navigation at best because on all other pages the focus order is always left button before the right button. If its not 3.2.3, they believe it would be usability.

I don't think this is a failure personally. Since WCAG says: 
> Focus order does not necessarily need to follow the visual layout of the web page, as long as the order in which elements receive focus is logical, and the hierarchy and relationship of content implied by the visual presentation is preserved.

**Comment by patrickhlauke** [comment]
doesn't fail normatively for me. don't think you can unambiguously argue that the focus order is illogical as such the way it is.

**Comment by TestPartners** [comment]
> doesn't fail normatively for me. don't think you can unambiguously argue that the focus order is illogical as such the way it is.

Agreed. Furthermore, I don't regard those buttons as being "navigation", but I am sure some people would argue that all links and buttons that take you somewhere else are "navigation". Maybe that needs to be clarified too.

**Comment by A11yTea** [comment]
Excellent, thank you everyone. I got the answer I was looking for.
